NoteSalihMVVM/Scenes/NoteList/NoteListViewController.swift

```diff
@@ -0,0 +1,273 @@
+//
+//  NoteListViewController.swift
+//  NoteSalihMVVM
+//
+//  Created by Mehmet Salih ÇELİK on 23.02.2022.
+//
+
+import TinyConstraints
+import MobilliumBuilders
+
+final class NoteListViewController: BaseViewController<NoteListViewModel> {
+    
+    private let refreshControl = UIRefreshControl()
+    private let addNoteButton = AuthButton()
+    private let noteTableView = UITableViewBuilder()
+        .build()
+    
+    var filteredItems: [NoteTableViewCellProtocol] = []
```

Comment:
viewModelde tutulup search ordan kontrol edilmeli, reload için clsoure tutulabilir